st2chatops/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM ubuntu:xenial
+
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG ST2_VERSION
+RUN : "${ST2_VERSION:?Docker build argument needs to be set and non-empty.}"
+
+LABEL maintainer="StackStorm <info@stackstorm.com>"
+LABEL com.stackstorm.vendor="StackStorm"
+LABEL com.stackstorm.support="Community"
+LABEL com.stackstorm.version="${ST2_VERSION}"
+LABEL com.stackstorm.name="StackStorm K8s HA"
+LABEL com.stackstorm.description="Docker image, optimized to run StackStorm \
+components and core services with Highly Available requirements in Kubernetes environment"
+LABEL com.stackstorm.url="https://stackstorm.com/#product"
+
+ENV container docker
+ENV ENV /etc/skel/.profile
+ENV TERM xterm
+
+# Default, but overrideable env vars to be substituted in st2.template chatops.conf
+ENV ST2_AUTH    http://st2auth:9100/
```

Comment:
I see now the inconsistency here.

Might be a good idea for us to change the ENV var name to `ST2_AUTH_URL` in `st2chatops` itself 

https://github.com/StackStorm/st2chatops/blob/c99d779d6e67385156bfd11b4c9997cf7e6f7ca8/st2chatops.env#L22-L24

Reply:
First change to rectify this: https://github.com/StackStorm/hubot-stackstorm/pull/163

Reply:
Second change in `st2chatops`: https://github.com/StackStorm/st2chatops/pull/119

Reply:
```suggestion
ENV ST2_AUTH_URL    http://st2auth:9100/
```